CollapseLauncher/XAMLs/MainApp/MainPage.Background.cs

```diff
@@ -0,0 +1,197 @@
+using CollapseLauncher.Helper.Background;
+using CollapseLauncher.Helper.Image;
+using CollapseLauncher.Helper.Metadata;
+using Hi3Helper;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using static CollapseLauncher.InnerLauncherConfig;
+using static CollapseLauncher.Statics.GamePropertyVault;
+using static Hi3Helper.Logger;
+using static Hi3Helper.Shared.Region.LauncherConfig;
+
+// ReSharper disable CheckNamespace
+// ReSharper disable RedundantExtendsListEntry
+// ReSharper disable InconsistentNaming
+// ReSharper disable UnusedMember.Local
+// ReSharper disable SwitchStatementHandlesSomeKnownEnumValuesWithDefault
+// ReSharper disable IdentifierTypo
+// ReSharper disable AsyncVoidMethod
+// ReSharper disable StringLiteralTypo
+// ReSharper disable CommentTypo
+// ReSharper disable SwitchStatementMissingSomeEnumCasesNoDefault
+
+namespace CollapseLauncher;
+
+public partial class MainPage : Page
+{
+    private static void BackgroundImg_IsImageHideEvent(object sender, bool e)
+    {
+        if (e) CurrentBackgroundHandler?.Dimm();
+        else CurrentBackgroundHandler?.Undimm();
+    }
+
+    private readonly HashSet<string> _processingBackground = [];
+    private async void CustomBackgroundChanger_Event(object sender, BackgroundImgProperty e)
+    {
+        if (_processingBackground.Contains(e.ImgPath))
+        {
+            LogWriteLine($"Background {e.ImgPath} is already being processed!", LogType.Warning, true);
+            return;
+        }
+
+        try
+        {
+            _processingBackground.Add(e.ImgPath);
+            var gameLauncherApi = LauncherMetadataHelper.CurrentMetadataConfig?.GameLauncherApi;
+            if (gameLauncherApi == null)
+            {
+                return;
+            }
+
+            gameLauncherApi.GameBackgroundImgLocal = e.ImgPath;
+            IsCustomBG                             = e.IsCustom;
+
+            // if (e.IsCustom)
+            //     SetAndSaveConfigValue("CustomBGPath",
+            //                           gameLauncherApi.GameBackgroundImgLocal);
```

Comment:
I think we can remove unused code comments like this one as it's no longer being used.

Reply:
Done